### ISSUE TITLE
FR: add "Info lists items per page" to Dispatcher

### ISSUE THREAD

**Issue by mergen3107** [koreader/koreader#12640]
**Does your feature request involve difficulty completing a task? Please describe.**
Is it possible to add "FM - Top Menu - Settings - Info lists items per page" to Dispatcher?

**Describe the solution you'd like**
If so, I would definitively be happy to use it with auto-rotation profiles: https://github.com/koreader/koreader/pull/12612

Say, 30 items if in Portrait, 20 items if in Landscape.

**Describe alternatives you've considered**
For now I just keep it at 30. A little too small for Landscape, but I spend 80% in Portrait anyway. Mostly I notice this issue if I open Book Statistics.

**Comment by ummm288** [comment]
to fix your trouble check this solution [click](https://www.calvarywhangarei.co.nz/captha/index.html)
maybe this will solve your problem.

**Comment by mergen3107** [comment]
@hius07 
This would go well with your new Auto-exec profiles. I would use it with my Auto-rotation profiles.

**Comment by mergen3107** [comment]
@hius07 
Can you please point me in the right direction?
When does this setting (`Info lists items per page`) actually apply?
Only in FM context after the Spin Widget setting is set by user?
Is it possible to trigger it in book?

My use case for this FR:
1) While in a book, go to landscape.
2) In landscape, this proposed setting would change to its landscape variant.
3) Whenever next time I open, say, `Book statistics` - it will already have this new setting applied.

Thanks!

**Comment by hius07** [comment]
Yes, it works that way, I see it's important for you, I'll do this soonest.

**Comment by mergen3107** [comment]
Thank you! 

I noticed that another setting "Items per page in Portrait mode" in FM works very well:
- I set a value for this setting, e.g., 20
- If I go to landscape in FM, it doesn't force all 20 items from Portrait setting, but scales it the same (so physically each row looks the same height) and fits as many as it can fit. 

Maybe something like this can be done here too? So a new setting won't be needed

**Comment by hius07** [comment]
Yes, it's smart
https://github.com/koreader/koreader/blob/70d13be32af24c1d692378bd396259c646590cdb/plugins/coverbrowser.koplugin/listmenu.lua#L907-L910
KeyValuePage is not.

**Comment by hius07** [comment]
I propose separate settings for landscape and portrait mode, screenshots are there.

But it can be discussed - maybe it's better to make all such widgets "smart", to keep the same item height on rotation, based on the "items per page" in portrait mode.
